src/pyroscope.rs

```diff
@@ -180,15 +181,15 @@ impl PyroscopeAgentBuilder {
         // Initiliaze the backend
         let backend = Arc::clone(&self.backend);
         backend.lock()?.initialize(self.config.sample_rate)?;
-        log::trace!("PyroscopeAgent - Backend initialized");
+        log::trace!(target:  LOG_TAG, "Backend initialized");
```

Comment:
there're 2 spaces before LOG_TARGET here and below
```suggestion
        log::trace!(target: LOG_TAG, "Backend initialized");
```